### PR TITLE
Feature/main runner

### PR DESCRIPTION
**Pull request by wendrickje**
dont merge yet, just showing you the progress
todo: refactor cleaner to use the dataframe from new week generator

### CODE REVIEW

File: modules/main.py [hunk 1]
```diff
@@ -1,26 +1,16 @@
 from machines import adaTree
 import sys
 from datetime import datetime
+import argparse
 
 if __name__ == '__main__':
-    if len(sys.argv) < 2:
-        print('Invalid number of parameters.')
-        exit()
+    parsed = argparse.ArgumentParser()
```
Comment (wendrickje) [inline modules/main.py]: good idea

File: modules/build_dataset.py [hunk 2]
```diff
@@ -36,25 +41,76 @@
...
+                    raise(e)
+
+
+# appends new week to master spreadsheet
+def add_new_week(year, week):
+    filename = get_filename(start_year, end_year)
+
+    backup_existing_master(filename)
+
+    with open(filename, 'a') as f:
```
Comment (wendrickje) [inline modules/build_dataset.py]: Try to avoid using f as a variable name as its actually a builtin for string formatting

Comment (ryanpmcintire) [reply modules/build_dataset.py]: Ahh good catch!

Comment (wendrickje) [reply modules/build_dataset.py]: no biggy just giving a tip for your day job

File: modules/cleaned.csv [hunk 3]
```diff
@@ -3358,20 +3358,20 @@
...
-6735,2022,2,phi,min,,1,-2.5,-0.0,0.0,0.0,25.4,18.9,23.3,16.0,1.2,1.4,1.8,1.3,6.959242991430959,7.554941302877522,7.040882528939539,6.592992931816461,5.720080914699335,3.7148268087526404,4.292997009870075,4.682434526728004,56.4,48.3,54.8,49.2,0.40923293473765865,0.3324382488239652,0.42125100318733055,0.2663138382216636,0.2790864028181304,-0.6387823524849616,1.5025477995151066,-0.9952769234004298,P
+6710,2022,1,ram,rai,W,-1,3.5,-5.0,24.0,19.0,24.4,23.5,27.4,23.6,1.0,0.8,1.3,1.3,6.230180557530534,6.472698824633613,6.557785640842707,6.561167447526238,4.136153021264066,4.330088546868113,5.094981938329236,4.8873470967582,46.5,43.0,43.1,49.0,0.3781866144509688,0.40931113464462154,0.33458005986129324,0.34627933325992355,-0.1911077065389511,0.1379578093582471,-0.6906589807791987,-0.7405679144007491,/boxscores/202209110sdg.htm
```
Comment (ryanpmcintire) [inline modules/cleaned.csv]: somethings not right here - taking a closer look

**Pull request merged**